characterEntities/YellowHero.java

```diff
@@ -86,4 +56,28 @@ protected boolean isHit(Ability ability, Entity target) {
 		}
 		return hit;
 	}
+
+	@Override
+	public void update() {
+		super.update();
```

Comment:
This update function looks pretty repetitive across all the hero types. Is there a reason why it was refactored this way?